bindata/v3.11.0/openshift-apiserver/deploy.yaml

```diff
@@ -115,17 +115,23 @@ spec:
         - mountPath: /var/log/openshift-apiserver
           name: audit-dir
         livenessProbe:
-          initialDelaySeconds: 30
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
         readinessProbe:
-          failureThreshold: 10
+          httpGet:
+            scheme: HTTPS
+            port: 8443
+            path: readyz
+          periodSeconds: 5
+        startupProbe:
           httpGet:
             scheme: HTTPS
             port: 8443
             path: healthz
+          failureThreshold: 150
```

Comment:
`1500s`  before the startup probe fails, when does the startup probe start ticking, when the container process starts running?